plugins/available/hosts.plugin.bash

```diff
@@ -0,0 +1,23 @@
+# Requested by philipjohn here : https://github.com/Bash-it/bash-it/issues/1264
+
+function hosts() {
```

Comment:
Please add the function meta information here, see here for an example: https://github.com/Bash-it/bash-it/blob/71ee7083cf2cef27b9af599dac2c655d47655b52/plugins/available/base.plugin.bash#L21-L24